src/overlays/actors/ovl_Bg_F40_Switch/z_bg_f40_switch.c

```diff
@@ -34,37 +35,160 @@ const ActorInit Bg_F40_Switch_InitVars = {
     (ActorFunc)BgF40Switch_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80BC4E04[] = {
+s32 sBgF40SwitchGlobalsInitialized = false;
+u32 sBgF40SwitchLastUpdateFrame;
+
+extern CollisionHeader D_06000118;
+extern Gfx D_06000438;
+extern Gfx D_06000390;
```

Comment:
This is new, so is not covered in the tutorial, but you are now supposed to look in the actor's object file (in this case object_f40_switch) in the `assets/objects/` folder for these, and replace the symbols by those generated for these offsets.

Of course you can also document the object: there's a few recent PRs you can follow to see what it should look like.